plugins/php/php_fpm_process

```diff
@@ -122,12 +164,12 @@ if ( defined $ARGV[0] and $ARGV[0] eq "config" )
 
   print <<"EOF";
 multigraph ${PLUGIN_NAME}_process
-graph_title php5-fpm processes for $pool
+graph_title php-fpm processes for $pool
 graph_args --base 1000 -l 0
 graph_vlabel Processes
 graph_scale yes
-graph_category processes
-graph_info This graph shows the php5-fpm process manager status from pool: $pool
+graph_category webserver
```

Comment:
I could imagine, that `appserver` would be more appropriate?

Reply:
Heh, PHP is so generic that I didn't think about that, but yeah, why not.